File: SISMUEBLUM JAVA/Persona.java
import java.util.*;
public class Persona extends  Cliente implements IConsultable
{
	private String nombre;
	private String apPat;
	private String apMat;
	private String dni;
	
	public Persona(){
		
	}
	public Persona(String nombre, String apPat,String apMat, String dni, String telefono, String correo, String direccion)             
	{
		super(telefono, correo, direccion);
		this.nombre = nombre;
		this.apMat = apMat;
		this.apPat = apPat;
		this.dni = dni;
	}


	public String getNombre (){
		return nombre; 
	}
	public void setNombre(String value){
		nombre = value; 
	}
	public String getApPat() { 
		return  apPat; 
	}
	public void setApPat (String value){
		apPat = value; 
	}
	public String getApMat() { 
		return apMat;
	}
	public void setApMat (String value){
		apMat = value; 
	}
	public String getDni() { 
		return dni;
	}
	public void setDni (String value){ 
		dni = value; 
	}

	public String consultarDatos()
	{
		return "";
	}

}

File: SISMUEBLUM C#/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


public class Persona : Cliente, IConsultable
{
	private string nombre;
	private string apPat;
	private string apMat;
	private string dni;

	public Persona(){
		
	}
	public Persona(string nombre, string apPat,string apMat, string dni, string telefono, string correo, string direccion) : 
		base(telefono, correo, direccion)
	{
		this.nombre = nombre;
		this.apMat = apMat;
		this.apPat = apPat;
		this.dni = dni;
	}


	public string getNombre (){
		return nombre; 
	}
	public void setNombre(string value){
		nombre = value; 
	}
	public string getApPat() { 
		return  apPat; 
	}
	public void setApPat (string value){
		apPat = value; 
	}
	public string getApMat() { 
		return apMat;
	}
	public void setApMat (string value){
		apMat = value; 
	}
	public string getDni() { 
		return dni;
	}
	public void setDni (string value){ 
		dni = value; 
	}

	public string consultarDatos()
	{
		throw new NotImplementedException();
	}

}

